### DISCUSSION TITLE
Static Connection to Databases

### DISCUSSION THREAD

**Discussion by ashwani-debug** [jobrunr/jobrunr#965] [General]
I have a Connection Object which gets the connection when the application starts. Now I want to use that connection everywhere in the jobrunr dashboard. How can I do that?

**Comment by rdehuyss** [comment]
I'm afraid JobRunr requires a `DataSource` and cannot work with only a sql `Connection`.

P.s.: we're a big fan of open-source etiquette which means your Github Profile should contain your name and preferably LinkedIn profile.
